README.md

```diff
@@ -42,6 +42,29 @@ To get a local copy up and running follow these simple example steps.
 ### Usage
 
 - Personal Project - Create a script file to house our methods using Ruby in the best approach in our opinion . 
+### Automated Tests
+
+- In root directory copy this:
+            ```
+            cd spec
+            ```
+- In your terminal copy this code to test the methods:
+            ```
+            rspec enumerable_spec.rb
+            ```
+- If one of the method is not working it will show you this error example:
```

Comment:
- [ ] There is a grammatical error in this line. This should be "If one of the **methods** is not working..."